simcon_project/templates/conversation/conversation_end.html

```diff
@@ -30,6 +30,8 @@ <h3 style="text-align:right;">{{ request.user }}</h3>
                     <input type="text" name="{{ response.id }}" id="{{ response.id }}"><br>
                 </div>
             {% endfor %}
+            <label for="{{ ct_response.self_rating }}">Rate how you did from 1 to 10: </label>
+            <input type="number" name="{{ ct_response.self_rating }}" id="{{ ct_response.self_rating }}" min="0" max="10"><br>
```

Comment:
I think the choices are suppose to be 1 to 5 and each rating should have a string associated to it, for example 5 would be "very good" or something similar to that. It's in the requirements document if you need a reference. Also I think how to choose the rating should be a radio button, both for clarity and and so the user can't input that aren't in between 1 to 5

Reply:
That can be done, but now I gotta look at how radio buttons work again lol

But the numbers can be limited rather easily with min and max tags

Reply:
The radio button is just a suggestion so I don't mid whichever way it is. We should get other people's opinion too